packages/asl-projects/client/project-router.js

```diff
@@ -216,6 +217,10 @@ const ProjectRouter = () => {
 
         <Switch>
           <Route path="/protocol-summary" component={ProtocolSummary} />
+          <Route path="/standard-protocol" render={(props) => (
+              <StandardProtocols {...props} />
+            )}
+          />
```

Comment:
I'd suggest not including the route if the flag is disabled
```suggestion
          { standardProtocolsEnabled
              ? <Route 
                  path="/standard-protocol" 
                  render={(props) => (<StandardProtocols {...props} />)}
                />
              : null
          }
```